### DISCUSSION TITLE
Creatures Waypoints

### DISCUSSION THREAD

**Discussion by virgo77** [azerothcore/azerothcore-wotlk#17965] [Guides & Tips]
Hi there, I need some help on how to put waypoints for multiple creatures at the same time. I explain: actually, in Mulgore (and in other zones too), there are some creatures that normally move together. Like Kodos or Galak. They are by 2, and move together in the Blizzard Classic servers.

I saw on the [GM Commands](https://www.azerothcore.org/wiki/gm-commands) page some commands that can add waypoints, and others that can 'link'. But, I'm not sure that 'linking' is between 2 creatures. I don't really understand all the comments in this tip: _.npc set link_ 

For 1 creature, no pb. I can make it moving as I want. But with 2 creatures, I want them to walk side by side, and still 'linked' if they are attacked. Actually I tried to do this but when I attack one of them, the second creature continues to walk.

So, can you help me about the commands I have to launch in ascending order, and make creatures to move/walk together, as on the Classic WotLK?

Thanks a lot :)

**Comment by Nyeriah** [comment]
You need to use `creature_formations` to add the follow and attack behavior you mention. You can't do waypoints properly with commands alone unfortunately.

**Comment by virgo77** [comment]
ok, so this one: ".npc add formation $leader"
How about "$leader"? I wonder it is the DBGUID of the creature I want to lead the group?!

maybe commands like these ones:
-- .npc add formation DBGUID1 DBGUID2 DBGUID3 $leader DBGUID99
-- .wp add DBGUID99
-- .wp reload DBGUID99
-- .wp load
and so the formation is moving, following DBGUID99?
